src/main/java/frc/robot/subsystems/intake/pivot/PivotState.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.subsystems.intake.pivot;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public enum PivotState {
+
+	ON_FLOOR(Rotation2d.fromDegrees(0)),
+	UP(Rotation2d.fromDegrees(1));
```

Comment:
Maybe in and out?

Reply:
What is in? When intake? when in robot?

Reply:
Personally I think the current names are good

Reply:
Yeah ur right